### ISSUE TITLE
Portal dies when restarting after reboot

### ISSUE THREAD

**Issue by yveskerwyn** [Jumpscale/jscockpit#396]
The [Cockpit Portal on `be-scale-1`](https://be-scale-1.demo.greenitglobe.com/CBGrid/Virtual%20Machine?id=352) failed to restart after a  reboot with following error:

```
load actor /opt/jumpscale8/apps/portals/portalbase/system/system__atyourservice/
'utf-8' codec can't decode byte 0xf3 in position 16: invalid continuation byte
```

It turned out to be related to MongoDB

For more details check with @nicolas_krassas (Telegram)

**Comment by rkhamis** [comment]
Verify/fix on 8.2.0

**Comment by ashraffouda** [comment]
can not reproduce
